.circleci/config.yml

```diff
@@ -136,6 +136,9 @@ _commands:
             name: Test Workspace | << parameters.workspace >>
             working_directory: << parameters.workspace >>
             command: |
+              apk --no-cache add python3
```

Comment:
because I need pyzmq library in the testing environment. This is needed in the system_test to check if there gets stuff published when expected.. :)

I hope this is the best way to add this.
I took the pip part actually from the navigation.ros.org git. So should be fine, but we cannot see if it works as the build process fails with nav2_waypoint_follower. But the install process / setup process of the docker seems to already worked fine. It just did not get to the point to use the system_test..

Reply:
You need to add it to the package.xml as a rosdep key. If the key doesn't exist, you must add it to rosdep to be merged. This would completely break the build farm.

Reply:
Can you support me here? Would be great to get this last thing before merging it. The rest can be done be me.

Reply:
I think this is it https://github.com/ros/rosdistro/blob/master/rosdep/python.yaml#L5350

Reply:
add to package.xml and remove these changes

Reply:
I added it as `  <test_depend>python-zmq</test_depend>` , I hope I got it right?

Reply:
I think it needs to be a depend, test depend means its only a dependency for running tests, but in fact you need it to run the feature at run-time, so it should be a `depend` so its installed on installation, not just in CI testing.

Reply:
are you sure? The feature out of my perspective is the Bt.CPP library stuff that is free of this python library.
This library is only needed for testing the BT.CPP feature (including the cpp-zmq stuff). (system-test)